gravitee-am-service/src/main/java/io/gravitee/am/service/authentication/crypto/password/Argon2IdPasswordEncoder.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.service.authentication.crypto.password;
+
+import org.springframework.security.crypto.argon2.Argon2PasswordEncoder;
+
+/**
+ * Wraps the Spring argon2 encoder.
+ */
+public class Argon2IdPasswordEncoder implements PasswordEncoder {
+
+    private final Argon2PasswordEncoder argon2PasswordEncoder = new Argon2PasswordEncoder();
```

Comment:
IHMO, we should make the configuration the values for the Argon2PasswordEncoder to give some flexibilitty to the admin and set by default the [OWASP recommendation](https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html).   

> Use [Argon2id](https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#argon2id) with a minimum configuration of 15 MiB of memory, an iteration count of 2, and 1 degree of parallelism.